memcache.py

```diff
@@ -67,6 +68,8 @@
 
 def cmemcache_hash(key):
     return (((binascii.crc32(key) & 0xffffffff) >> 16) & 0x7fff) or 1
+
```

Comment:
🤔

Reply:
Correction to [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines), which requires two blank lines after function declaration.

Also, it fixed the CI which is running `flake8`.